csharp-advanced-topics/DelegatesInCsharp/DelegatesInCsharp.sln

```diff
@@ -1,11 +1,15 @@
 ﻿

```

Comment:
Not sure what happened here, but you seem to have used one of our already existing projects. You need to create your own solution and not use the existing ones. Each solution in our repository is one article that's published. If you do it like this you can break the existing code.